Makefile

```diff
@@ -86,7 +85,7 @@ linters: lint generate-check fmt-check
 
 .PHONY: lint
 lint:
-	$(GOLANGCI_LINT) run
+	IS_CONTAINER=true ./hack/golint.sh
```

Comment:
This line should not set IS_CONTAINER. The reason for having IS_CONTAINER in the scripts is so a developer can run the tools in containers locally to avoid conflicts if we need a different version of a tool than another project needs.